src/store/collections.ts

```diff
@@ -0,0 +1,116 @@
+import Vue from 'vue';
```

Comment:
Why do you think we need an specific store for this and not the items store? 

The item state handling still needs a huge refactor, our store design needs a refactor, which is still on hold: https://github.com/jellyfin/jellyfin-vue/issues/801